### PR TITLE
Convert vendor and package to camel case when installing package from git

### LINKED ISSUE #65: Convert Pascal case to Kebab case
Hi,

is it possible to have a package name like MyAwesomePackage that becomes my-awesome-package rather than myawesomepackage?

### LINKED ISSUE #71: Incorrect assumption of ServiceProvider class name using git install
Using the `packager:git` command, url segments are used to create the service provider class entry in the `config/app.php` file. This will most likely need to incorrect registering of plugin service providers. For example, running the command:

    php artisan packager:git https://github.com/Jeroen-G/laravel-packager

Will result in the following being added to `config/app.php`:

    jeroen-g\laravel-packager\laravel-packagerServiceProvider::class

rather than:

    JeroenG\Packager\PackagerServiceProvider::class

Although `vendor` and `package` can be passed as parameters, running this command without these arguments does not automatically put the default `vendor` and `package` into title case.

Secondly, in the event of the user installing a package with a hyphen, there is no prompt to run the command with the `vendor` and `package` arguments.

### PR DESCRIPTION
**Pull request by kakposoe**
Closes #71, Closes #65.

This formats the `vendor` and `package` when using the `php artisan packager:git` command.

Using  `->formatVars()` method, both are changed to title case. If there are any hyphens, the string is split, changed to title case and concatenated.

### CODE REVIEW

**Comment by Jeroen-G** [comment]
Thanks for working on all this!
For any future PR's, try creating separate branches or rebasing with the origin. Because part of the commits mentioned here are from the previous PR (no problem for now by the way!)

File: src/Wrapping.php [hunk 1]
```diff
@@ -135,4 +139,22 @@ public function removeFromProviders($vendor, $package)
         return $this->replace($vendor.'\\'.$package.'\\'.$package.'ServiceProvider::class,', '')
                     ->fillInFile(config_path('app.php'));
     }
+
+    /**
+     * Format vendor and package strings.
```
Comment (Jeroen-G) [inline src/Wrapping.php]: Maybe you could be more precise as to what/how it formats?

**Comment by kakposoe** [comment]
> Thanks for working on all this!
> For any future PR's, try creating separate branches or rebasing with the origin. Because part of the commits mentioned here are from the previous PR (no problem for now by the way!)

I will do so in the future. Will make change to `formatVars` method doc shortly.

**Comment by bilogic** [comment]
Hi, can share why there is a need to camel case vendor and package? It is causing a discrepancy in my class names and folder names, i.e. folder names are all small case but class names are camel case. Thank you.

**Pull request merged**